netmiko/base_connection.py

```diff
@@ -534,6 +567,13 @@ def establish_connection(self, width=None, height=None):
         if self.protocol == 'telnet':
             self.remote_conn = telnetlib.Telnet(self.host, port=self.port, timeout=self.timeout)
             self.telnet_login()
+        elif self.protocol == 'serial':
+            self.serial_settings["port"] = str(self.port).split(" ")[0]
```

Comment:
See earlier comment on overloading of port (i.e. I think we should make it a distinct attribute like `comm_port` (or something like that).